gradle/libs.versions.toml

```diff
@@ -87,6 +90,8 @@ kotlin-test = { module = "org.jetbrains.kotlin:kotlin-test-junit", version.ref =
 kotlinpoet = { module = "com.squareup:kotlinpoet", version.ref = "kotlinpoet" }
 kotlinpoet-ksp = { module = "com.squareup:kotlinpoet-ksp", version.ref = "kotlinpoet" }
 
+kgx = { module = "com.rickbusarow.kgx:kotlin-gradle-extensions", version.ref = "kgx" }
```

Comment:
Sneaky

Reply:
Hiding in plain sight!

Actually, I forgot about this or I would have called it out.  It has lots of excellent build-logic yak-shaving applications.